man/journald.conf.xml

```diff
@@ -348,7 +348,10 @@
         <literal>notice</literal> for <varname>MaxLevelKMsg=</varname>,
         <literal>info</literal> for <varname>MaxLevelConsole=</varname>,
         and <literal>emerg</literal> for
-        <varname>MaxLevelWall=</varname>.</para></listitem>
+        <varname>MaxLevelWall=</varname>. These settings may be
```

Comment:
"These" would mean MaxLevelKMsg=, MaxLevelConsole=, MaxLevelWall=. So this needs to be reworded.

Or preferably, just add systemd.journald.max_level_kmsg, systemd.journald.max_level_wall to keep things consistent.